internal/pkg/flink/config/color.go

```diff
@@ -1,6 +1,15 @@
 package config
 
-import "github.com/confluentinc/go-prompt"
+import (
+	"github.com/fatih/color"
 
-// We should use this package as a root for coloring variables
-const HighlightColor = prompt.Cyan
+	"github.com/confluentinc/go-prompt"
+)
+
+// Package used as a root for color used across the application
+const PromptAccentColor = prompt.Cyan
+const AccentColor = color.FgCyan
+
+const InfoColor = color.FgWhite
+const ErrorColor = color.FgHiRed
+const WarnColor = color.FgCyan
```

Comment:
Can you move these constants into `pkg/color`? We've been talking about doing this for the whole CLI, and these would be the colors we need.

Reply:
> Erros should be red, info white and and warnings yellow.

This doesn't seem consistent by the way- did you mean to use yellow instead of cyan?

Reply:
Ops, I typed the wrong color. Thanks